### ISSUE TITLE
Some sections and fields do not use device font size or do not adapt to it

### ISSUE THREAD

**Issue by shanaqui** [HabitRPG/habitrpg-ios#950]
This might be better split into two separate issues, one for where the font size isn't correct, and one where the app doesn't adapt, but I'll just get it in Github for now!

User report:

> Here’s what what I see when I click on “About”:
> 
> ![image](https://user-images.githubusercontent.com/8144640/98802823-c4cde480-240b-11eb-8da3-2585d23aa165.png)
> 
> Some of those buttons I can’t even click on because they overlap- specifically the report a bug button, I had to decrease the font on my device and use magnification to find it. 
> I am also realizing that some of the pages in this menu don’t use the device’s set font size.
> 
> ![image](https://user-images.githubusercontent.com/8144640/98802872-dadba500-240b-11eb-9d95-1ebfe8d8b6aa.png)
>
>  The main issue is tasks:
> 
> ![image](https://user-images.githubusercontent.com/8144640/98803012-0e1e3400-240c-11eb-859a-6b3b360c449f.png)
>
> The only thing that doesn’t work well for me on this page is just that subtasks get cut off, but this problem is amplified in the habit creation page:
>
> ![image](https://user-images.githubusercontent.com/8144640/98802909-eb8c1b00-240b-11eb-9015-6b80eef81ec8.png)
> 
> For the sub tasks, it’s hard to continue writing if you can’t see the whole subtask, so I think this could be solved by having the text wrap around.
>
> The biggest issue is that the font size for when you are setting the title and description of the task don’t adjust to the my device’s set font size (it’s really, really small in the picture above). I think the habit and daily creation have the same problem.

**Comment by saraolson** [comment]
dang the about text is huge in this.. not sure how much of an undertaking fixing all this would be but the screenshots are really helpful in pointing out specific areas to focus on. will see if we can start chipping away at these in next milestone

**Comment by saraolson** [comment]
think the only ones still needing fixed are the about screen and the main header

**Comment by saraolson** [comment]
this is fixed now
